pgzero/game.py

```diff
@@ -243,7 +250,7 @@ def mainloop(self):
 
         pgzclock = pgzero.clock.clock
 
-        self.need_redraw = True
+        self.need_redraw = not paused
```

Comment:
I don't think this is right - this is at the start of the game while the pause state will change during the game loop.